README.md

```diff
@@ -4,49 +4,56 @@ A django site for kinetic models
 ![build](https://github.com/comocheng/kineticmodelssite/workflows/Run%20Tests/badge.svg)
 
 
-## Setup
-Install Dependencies:
-To get a virtual environment up and running, [install an Anaconda distribution](https://www.anaconda.com/products/individual), then run:
+## Development Setup
+Clone the project and setup [docker](https://www.docker.com/products/docker-desktop) and [docker-compose](https://docs.docker.com/compose/install/).
 
-```conda env create -f environment.yml```
+For the first time, you need to run the initial migrations, which use [RMG-models](https://github.com/comocheng/RMG-models).
+To do this, first clone the RMG-models repo via ssh or https (ssh shown below):
 
-Then set up PostgreSQL:
-- ```$ conda install postgresql psycopg2```
-- ```$ initdb -D postgres```
-- ```$ pg_ctl -D postgres -l postgres.log start```
-- ```$ createuser --interactive --pwprompt``` then add user `postgres` password `postgres` as superuser (y).
-- ```$ psql postgres``` then `\du` to check the `postgres` Role is defined, then `\q` to quit.
-- ```$ python manage.py migrate`
+```git clone git@github.com:comocheng/RMG-models.git```
 
+Now create a named volume and copy the files over using `docker`:
 
-## Quickstart:
-To get a local version of the site up and running, run these commands in order:
-- ```conda activate kms_env```
-- ```python manage.py migrate```
-- ```python manage.py runserver```
+```docker volume create rmg-models```
 
-The terminal will then output which address and port the site is hosted on.
+```
+docker run --rm \
+--volume=/path/to/RMG-models:/source \
+--volume=rmg-models:/target \
+busybox cp -av "/source/." "/target/"
+```
+
+Then run:
+
+```docker-compose up```
```

Comment:
After `docker-compose up` on my iMac I got
```
<bunch of stufff....>
Removing intermediate container fa267800deea
 ---> 772c6fc54efd
Step 6/15 : COPY environment.yml /app/
 ---> 78662334a779
Step 7/15 : RUN mkdir /kms_env
 ---> Running in 050302c86428
Removing intermediate container 050302c86428
 ---> f43796cdf7c7
Step 8/15 : ENV ENV_PREFIX=/kms_env
 ---> Running in a12938717aaa
Removing intermediate container a12938717aaa
 ---> 4e5e92234168
Step 9/15 : RUN conda env create --prefix $ENV_PREFIX --file environment.yml --force
 ---> Running in 2fdb902e9221
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... ERROR: Service 'web' failed to build : The command '/bin/bash -c conda env create --prefix $ENV_PREFIX --file environment.yml --force' returned a non-zero code: 137
```
Looks like the conda step failed.

Reply:
googling it, seems 137 suggests not enough memory.

Reply:
or not... https://stackoverflow.com/questions/57291806/docker-build-failed-after-pip-installed-requirements-with-exit-code-137

Reply:
I increased the memory given to docker using the Docker Desktop app (from 2GB to 4GB) and so far it seems to be working. 
<img width="1322" alt="Screen Shot 2020-10-15 at 3 41 05 PM" src="https://user-images.githubusercontent.com/93807/96178130-edda8280-0efc-11eb-85a9-54a247ecba14.png">

Reply:
```suggestion
```docker-compose up```

If you get an error ending `...returned a non-zero code: 137` then probably your Docker needs more RAM. On MacOS or Windows you can configure this using your Docker Dashboard app settings. In our experience 2GB is not enough but 4GB is.
```